src/shellbar.scss

```diff
@@ -75,45 +62,78 @@ $block: #{$fd-namespace}-shellbar;
     &--copilot {
       justify-content: center;
       order: 2;
+
+      .#{$block}__button {
+        padding-left: 0;
+      }
     }
 
     &--actions {
       justify-content: flex-end;
       order: 3;
     }
+
+    .#{$block}__product-menu {
+      cursor: pointer;
+      color: $fd-shellbar-color;
+      background: $fd-shellbar-background-color;
+
+      &:hover {
+        background: $fd-shell-hover-background;
+      }
+
+      &:active,
+      &:focus {
+        background: $fd-shell-active-background;
+        color: $fd-shell-active-text-color;
+      }
+    }
+
+    .#{$block}__identifier--circle {
+      background-color: var(--sapAccentColor6);
+    }
   }
 
   &__button {
     @include fd-button-reset();
     @include fd-icon-size("m", "before");
     @include fd-icon-size("m", "after");
-    @include fd-var-color("color", fd-color("shell", 2), --fd-color-shell-2);
 
     border-color: transparent;
+    background: $fd-shellbar-background-color;
+    color: $fd-shellbar-color;
 
-    @include fd-active-pressed-selected() {
-      background-color: hsl(217, 21%, 21%);
+    &:hover {
+      background: $fd-shell-hover-background;
     }
 
-    @include fd-hover() {
-      background-color: hsl(217, 21%, 25%);
+    &:active {
```

Comment:
Shouldn't we use `@fd-hover`,  `@fd-active`... mixins here? I'm not sure so that's rather the question for more experienced ones with styles (@InnaAtanasova). Anyway I think we should choose one approach, using mixin or explicit selectors.